html/elecnorms.html

```diff
@@ -136,6 +136,11 @@
                     <div class="stuff col-12 outline">
                         <div class="con innerStuff">
                             <embed class="pdf" type="application/pdf" src="../assets/pdfs/elecnorms.pdf"></embed>
+                            <a href="../assets/pdfs/elecnorms.pdf">
```

Comment:
This looks good. Should we also do this for the constitution and change the pages here to close #6?

Reply:
This just adds the download button to the document. I can add it to the constitution page as well. #6 deals more with merging the two pages which I haven't looked into yet.

Reply:
I think we can leave the constitution for when we do #6 then.

Reply:
Sounds good, we'll do it in a separate PR then.